src/Components/Atoms/HomeButton/HomeButton.tsx

```diff
@@ -0,0 +1,8 @@
+import { HomeButtonProps } from "./HomeButtonPropsInterface";
```

Comment:
What is the diffference between Auth Button & Home Button ?
Can only one component can be used for both of them ?